third_party/stdlib/dummy_thread.py

```diff
@@ -0,0 +1,145 @@
+"""Drop-in replacement for the thread module.
```

Comment:
This is good to have since it seems to be used by some other modules, but I think it should be pretty straightforward to implement a thread.py. See threading.py for some examples.